src/providers/seventv/SeventvEmotes.hpp

```diff
@@ -43,6 +41,7 @@ class SeventvEmotes final
     void loadEmotes();
     static void loadChannel(std::weak_ptr<Channel> channel,
                             const QString &channelId,
+                            const QString &channelDisplayName,
```

Comment:
Why do we need display name now? Keep in mind that this can be non-ascii characters in some cases, and if anything - login name should be used instead of display name. Again, I think this is an unnecessary argument.

Reply:
As mentioned above:
> Didn't realize there was ID support (only looked at the docs). I will switch it to id so things are cleaner.

Reply:
As long as it's removed with the change back to IDs I'm happy.